### PR TITLE
Added documentation for adding Docker build args.

### PR DESCRIPTION
**Pull request by oatmealraisin**
Short section on Build Args. @bparees ptal?

Waiting on [the related PR in Origin](https://github.com/openshift/origin/pull/12439)

### CODE REVIEW

**Comment by bparees** [comment]
lgtm, assuming of course that you get api approval for the api shown in the example :)

File: dev_guide/builds.adoc [hunk 1]
```diff
@@ -538,6 +538,27 @@ endif::[]
 You can also manage environment variables defined in the `BuildConfig` with the
 xref:../dev_guide/environment_variables.adoc#dev-guide-environment-variables[`oc set env`] command.
 
+[[docker-strategy-build-args]]
+=== Docker Build Arguments
+
+
```
Comment (ahardin-rh) [inline dev_guide/builds.adoc]: Just remove one extra line here. The rest LGTM! :beers:

**Comment by oatmealraisin** [comment]
[(5) Support dockerfile ARGs in docker build [techdebt][builds]](https://trello.com/c/Yh9BPAGi/1051-5-support-dockerfile-args-in-docker-build-techdebt-builds)

**Comment by ahardin-rh** [comment]
@oatmealraisin Whoops! Heads-up that this will have to be rebased since the Builds topic was just [reworked into smaller chunks](https://docs.openshift.com/container-platform/3.4/welcome/revhistory_full.html#wed-feb-22-2017). Looks like you will want to move this new section to the Build Strategies file https://docs.openshift.com/container-platform/3.4/dev_guide/builds/build_strategies.html  :eye_speech_bubble:

**Comment by oatmealraisin** [comment]
@ahardin-rh Thanks for the heads up!

**Comment by bparees** [comment]
@oatmealraisin bump

**Comment by oatmealraisin** [comment]
@ahardin-rh Moved everything to Build Strategies, ptal?

@bparees The content didn't change.

File: dev_guide/builds/build_strategies.adoc [hunk 2]
```diff
@@ -323,6 +323,27 @@ console], set the *Source Secret*.
 
 . Click the *Save* button.
 
+
+[[docker-strategy-build-args]]
+=== Docker Build Arguments
+
+To set link:http://docs.docker.com/v1.7/reference/api/hub_registry_spec/#docker-registry-1-0[Docker build arguments]
+, add entries to the `BuildArgs` array, which is located in the
```
Comment (ahardin-rh) [inline dev_guide/builds/build_strategies.adoc]: Overall, this looks great! My one nit is that it looks like a weird line break here. Other than that, I think we're ready to go. Thanks!

Comment (oatmealraisin) [inline dev_guide/builds/build_strategies.adoc]: Sorry, I break on >80 characters, fixed

**Comment by ahardin-rh** [comment]
Thank you!

**Comment by ahardin-rh** [comment]
[rev_history]
|xref:../dev_guide/builds/build_strategies.adoc#dev-guide-build-strategy-options[Builds -> Build Strategy Options]
|Added a new section on adding xref:../dev_guide/builds/build_strategies.adoc#docker-strategy-build-args[Docker build arguments].
%

**Pull request merged**